src/main/kotlin/no/nav/syfo/infrastructure/kafka/KandidatStatusProducer.kt

```diff
@@ -0,0 +1,57 @@
+package no.nav.syfo.infrastructure.kafka
+
+import no.nav.syfo.application.IKandidatStatusProducer
+import no.nav.syfo.domain.SenOppfolgingKandidat
+import no.nav.syfo.domain.asProducerRecordKey
+import no.nav.syfo.util.configuredJacksonMapper
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.apache.kafka.common.serialization.Serializer
+import org.slf4j.LoggerFactory
+import java.time.OffsetDateTime
+import java.util.*
+
+class KandidatStatusProducer(private val producer: KafkaProducer<String, KandidatStatusRecord>) :
+    IKandidatStatusProducer {
+
+    override fun send(kandidatStatus: SenOppfolgingKandidat): Result<SenOppfolgingKandidat> =
+        try {
+            producer.send(
+                ProducerRecord(
+                    TOPIC,
+                    kandidatStatus.personident.asProducerRecordKey(),
+                    KandidatStatusRecord.fromSenOppfolgingKandidat(kandidatStatus),
+                )
+            ).get()
+            Result.success(kandidatStatus)
+        } catch (e: Exception) {
+            log.error("Exception was thrown when attempting to send vurdering: ${e.message}")
+            Result.failure(e)
+        }
+
+    companion object {
+        private const val TOPIC = "teamsykefravr.ismeroppfolging-kandidat-status"
+        private val log = LoggerFactory.getLogger(KandidatStatusProducer::class.java)
+    }
+}
+
+data class KandidatStatusRecord(
```

Comment:
Ser for meg at vi trenger et status-felt her etter hvert - i første omgang noe som sier om kandidaten har svart ja eller nei på om de trenger oppfølging, men etter hvert også om de har svart? Så må vel oversikten (og andre konsumenter) få informasjon når veileder har behandlet/vurdert kandidaten, men det kan vi jo utvide med senere. Kanskje det blir en egen topic :man_shrugging:

Reply:
Hmm ja, vi har kanskje ikke snakket helt om hva som burde sendes her. Med denne DTOen sender vi kandidat mer enn status. Vi snakket vel om å lagre ned enumen `trenger_oppfolging` i oversikten, men går kanskje bort fra det hvis vi heller vil kjøre "flagg+hent fra et API"-approachen. Vi kan diskutere den i morgen, og så kan en av oss ta over oppgaven?